app/header.scss

```diff
@@ -1,130 +1,276 @@
 @import '../assets/stylesheets/_custom-variables';
 
-$pf-cust-navbar-text-size:$font-size-large;
+// $pf-cust-navbar-text-size:$font-size-large;
 
-$pf-custom-navbar-top-border-width:2px;
-$pf-custom-navbar-header-border-width:0;
+// $pf-custom-navbar-top-border-width:2px;
+// $pf-custom-navbar-header-border-width:0;
 
-$pf-custom-navbar-item-height:$pf-cust-navbar-text-size*2;
-$pf-custom-navbar-persistent-nav-offset:$pf-custom-navbar-item-height+ $pf-custom-navbar-header-border-width;
-$pf-custom-navbar-font-size:$pf-cust-navbar-text-size;
-$pf-custom-navbar-icon-height:$pf-cust-navbar-text-size *1.5;
-$pf-custom-navbar-icon-min-width:$pf-cust-navbar-text-size*1.6;
-$pf-custom-navbar-item-padding:0;
+// $pf-custom-navbar-item-height:$pf-cust-navbar-text-size*2;
+// $pf-custom-navbar-persistent-nav-offset:$pf-custom-navbar-item-height+ $pf-custom-navbar-header-border-width;
+// $pf-custom-navbar-font-size:$pf-cust-navbar-text-size;
+// $pf-custom-navbar-icon-height:$pf-cust-navbar-text-size *1.5;
+// $pf-custom-navbar-icon-min-width:$pf-cust-navbar-text-size*1.6;
+// $pf-custom-navbar-item-padding:0;
```

Comment:
I've merged this, but please just delete un-needed code. We have the history in git if needed.